src/io/flutter/sdk/FlutterSdk.java

```diff
@@ -369,27 +339,12 @@ private static FlutterSdk getFlutterFromDartSdkLibrary(Library lib, VirtualFile
   }
 
   public enum Command {
-
-    CREATE("Flutter create", "create") {
-      @Override
-      void onStart(@Nullable Module module, @Nullable VirtualFile workingDir, @NotNull String... args) {
-        // Enable Dart.
-        if (module != null) {
-          ApplicationManager.getApplication()
-            .invokeLater(() -> ApplicationManager.getApplication().runWriteAction(() -> DartPlugin.enableDartSdk(module)));
-        }
-      }
```

Comment:
This sounds like a good thing to put in a test, once we have integration tests working.